test/e2e/infra/secondary_network/secondary_network_prerequisite.sh

```diff
@@ -0,0 +1,361 @@
+#!/bin/bash
```

Comment:
maybe we could just name this script `prepare_cluster.sh`?

Reply:
renamed file as prepare_cluster.sh